articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+---
+title: How to create a site-to-site connection using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Create a site-to-site (S2S) VPN gateway connection from an on-premises network to UKCloud for Microsoft Azure 
+services: azure-stack
+author: Shaion O'Niel
+reviewer: Daniel Brennand
+lastreviewed: 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a VPN connection
+toc_sub3:
+toc_sub4:
+toc_title: Create a site-to-site VPN connection between Azure Stack & Public Azure - portal
+toc_fullpath: Users/How To/azs-how-create-S2S-VPN-azure-to-azurestack.md
+toc_mdlink: azs-how-create-S2S-VPN-azure-to-azurestack.md
+---
+
+# How to create a site-to-site connection between Azure Stack and Public Azure using the UKCloud Azure Stack portal
+
+## Overview
+
+This article shows you how to use the Azure Stack and Public Azure portals to create a Site-to-Site VPN between your UKCloud for Microsoft Azure environment and Public Azure environment.
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in both the Azure Stack and Public Azure portals.
+
```

Comment:
As this is a long article, it might be worth including an extra section here that provides an overview of the process:
1. Set up VPN on the Azure Stack side
2. Set up VPN on the Public Azure side
3. Update local gateway IPs
4. Verify the VPN connection